labtool2.0/src/components/StudentTable.js

```diff
@@ -398,8 +406,9 @@ export const StudentTable = props => {
           selection
           style={{ width: `${getBiggestWidthInDropdown(dropDownFilterTeachers)}em` }}
         />
+        <span> Add filtering tag: </span>
+        <Dropdown scrolling options={dropDownTags} onChange={changeFilterTag()} placeholder="Select Tag" value="" selection style={{ width: `${getBiggestWidthInDropdown(dropDownTags)}em` }} />
```

Comment:
And this could be then 
```suggestion
        <Dropdown scrolling options={dropDownTags} onChange={changeFilterTag} placeholder="Select Tag" value="" selection style={{ width: `${getBiggestWidthInDropdown(dropDownTags)}em` }} />
```